modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -444,8 +456,11 @@ object EpicsUtil {
     get.map(_ =!= v.some).map(_.option(set))
 
   def smartSetDoubleParam(relTolerance: Double)(v: Double, get: => Option[Double], set: SeqAction[Unit]): List[SeqAction[Unit]] =
-    if(get.forall(x => (v === 0.0 && x =!= 0.0) || abs((x - v)/v) > relTolerance))
-  List(set) else Nil
+    if (get.forall(x => !(v === 0.0 && x === 0.0) || (v === 0.0 || abs((x - v)/v) > relTolerance))) {
```

Comment:
Same here, that is a `&&`

Reply:
`toleranceTest` ?

Reply:
Well it is just a name, some better suggestion?

Reply:
I was too laconic. I was not complaining about the name, but pointing that it should be used here.